juju/juju
# Classic snap

## Description of change

This change enables the easy building of classic snaps for juju, including bash completion support.

## QA steps

Verify the change works by creating your own local juju snap. Verify bash completion works, and juju sees your current environments, etc.

## Documentation changes
The current user workflow for juju isn't changed. For developers, building a snap against there local source tree now works out of the box nicely :-)

## Bug reference
